core/invehicle-digital-twin/src/invehicle_digital_twin_config.rs

```diff
@@ -15,8 +16,13 @@ pub struct Settings {
 
 /// Load the settings.
 pub fn load_settings() -> Settings {
+    let config_filename_path = match env::var("IBEJI_HOME") {
```

Comment:
Please provide details on the IBEJI_HOME environment variable in Ibeji's main README.